lib/wallaroo/startup.pony

```diff
@@ -853,9 +853,9 @@ actor Startup
 
     let file = File(worker_names_filepath)
     for worker_name in file.lines() do
-      ws.push(worker_name)
-      @printf[I32]("recover_worker_names: %s\n".cstring(),
-        worker_name.cstring())
+      let w: String val = consume worker_name
+      ws.push(w)
```

Comment:
Can you consume this directly in the push call?  If it doesn't work as
```
ws.push(consume worker_name)
```
try
```
ws.push(consume val worker_name)
```
but I would have thought the first would work.

Reply:
I would have, but then it needs to get printed in the line below, hence the assignment to `w`.